### DISCUSSION TITLE
Please add a load/save function to load and save the current selections

### DISCUSSION THREAD

**Discussion by BinaryRider** [undergroundwires/privacy.sexy#360] [Ideas]
Hello,

a great collection of scripts. It comes in handy as I am setting up a new Win11 computer.

BUT: It would be a great help if you could save your current selection and load it again.
I like to test a few settings first and if there are no negative results after a few days, I will use more scripts. Unfortunately, I always have to remember what I enabled or disabled the days before. 

Ultimately, at the end of the test phase (several weeks), I would like to have a suitable script that contains all the points that have proved useful for me. 

But as I said, I have to write down what I have already tried and whether this script was okay or not.

Please, please add a load/save function to load and save the current selection.

Thank you very much.
BR

**Comment by undergroundwires** [comment]
I understand and I agree. It's planned, see #59. Features are coming late and stability updates first. ID support will be rolled out in 0.14.0 that first that will enable this.
